src/node/install.sh

```diff
@@ -164,6 +164,13 @@ else
     fi
 fi
 
+# Install pnpm
+if type pnpm > /dev/null 2>&1; then
+    echo "pnpm already installed."
+else
+    npm install -g pnpm
```

Comment:
Can you help write a test to validate these changes? Thanks! 
Adding checks to existing scenarios should be fine too - https://github.com/devcontainers/features/tree/main/test/node

Reply:
I've added tests.